src/ray/protobuf/node_manager.proto

```diff
@@ -542,4 +562,7 @@ service NodeManagerService {
   // without completing outstanding work.
   // Failure: Retries, it's idempotent.
   rpc KillLocalActor(KillLocalActorRequest) returns (KillLocalActorReply);
+  // Forwards the CancelTask request from the caller core worker to the executor
+  // Failure: Retries, it's idempotent.
```

Comment:
The description here is really an abstraction leak. The caller doesn't need to know that the RPC is forwarded, just that if they get `OK`, the task is canceled.

Also would document `force` escalation policy.